tests/phpunit/tests/post/wpTrashPost.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+class Tests_Post_wpTrashPost extends WP_UnitTestCase {
+	/**
+	 * @var WP_Post
+	 */
+	protected $post;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->post = $this->factory()->post->create_and_get(
+			array(
+				'post_status' => 'draft',
+			)
+		);
+	}
+
```

Comment:
And a mention of the related ticket for each test case. For example:
```
	/**
	 * @ticket 58392
	 * @covers ::wp_trash_post
	 */
```

See https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/post/wpPublishPost.php